Core/Validation/CreateTripValidation.cs

```diff
@@ -22,19 +23,19 @@ public TripCreateValidation()
             RuleFor(offer => offer.Description)
                 .MinimumLength(1)
                 .MaximumLength(1000)
-                .WithMessage("The description must be between 1 and 1000 symbols!");
+                .WithMessage("'{PropertyName}' must be between 1 and 1000 symbols!");
```

Comment:
Is it ok that minimal description length only 1 symbol? What we need to put in the description to have one symbol?